docs/en/02_Developer_Guides/03_Forms/Field_types/03_HTMLEditorField.md

```diff
@@ -224,9 +224,9 @@ Example: Remove field for "image captions"
 
 	:::php
 	// File: mysite/code/MyToolbarExtension.php
-	class MyToolbarExtension extends Extension {
+	class MyToolbarExtension extends DataExtension {
```

Comment:
I put that because when searching for Extension in the API nothing comes back, only DataExtension.